README.md

```diff
@@ -4,49 +4,87 @@
 
 > A webpack loader that moves a module and its dependencies into a Web Worker, automatically reflecting exported functions as asynchronous proxies.
 
-- Bundles a tiny, purpose-built RPC implementation into your app
-- If exported module methods are already async, signature is unchanged
-- Supports synchronous and asynchronous worker functions
-- Works beautifully with async/await
-- Imported value is instantiable, just a decorated `Worker`
-
+* Bundles a tiny, purpose-built RPC implementation into your app
+* If exported module methods are already async, signature is unchanged
+* Supports synchronous and asynchronous worker functions
+* Works beautifully with async/await
+* Imported value is instantiable, just a decorated `Worker`
 
 ## Install
 
 ```sh
 npm install -D workerize-loader
 ```
 
-
 ### Usage
 
 **worker.js**:
 
 ```js
 // block for `time` ms, then return the number of loops we could run in that time:
 export function expensive(time) {
-    let start = Date.now(),
-        count = 0
-    while (Date.now() - start < time) count++
-    return count
+	let start = Date.now(),
+		count = 0;
+	while (Date.now() - start < time) count++;
+	return count;
 }
 ```
 
 **index.js**: _(our demo)_
 
 ```js
-import worker from 'workerize-loader!./worker'
+import worker from "workerize-loader!./worker";
 
-let instance = worker()  // `new` is optional
+let instance = worker(); // `new` is optional
 
-instance.expensive(1000).then( count => {
-    console.log(`Ran ${count} loops`)
-})
+instance.expensive(1000).then(count => {
+	console.log(`Ran ${count} loops`);
+});
 ```
 
+### About [TypeScript](http://typescriptlang.org)
+
+If you're using [TypeScript](http://typescriptlang.org) in your build you have to modify your
+webpack configuration as follows:
+
+```js
+module: {
+  rules: [
+    {
+      test: /\.worker\.ts$/,
+      use: ['workerize-loader', 'ts-loader'],
```

Comment:
is this correct? `ts-loader` should already be getting applied via line 59.

Reply:
this solutions not work for me

Reply:
> is this correct? ts-loader should already be getting applied via line 59.

I'm guessing that the purpose here is to ensure that `.worker.ts` are chained through ts-loader and then workerize-loader?

Reply:
To be honest I can't remember. It's over one year ago when I opened that pull request 😅 The only thing I can remember is that I had some issues and I copy and pasted my solution into this pull request and it worked.

Reply:
Do does the solution in your pr work for you? I'm thinking of taking it for a spin (I happened upon this in the first place as I'm planning to use web workers with TypeScript and webpack )

Reply:
The last time when I played around with that was 1 year ago. So I don't know if it still works 🤷‍♂️